src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -99,4 +100,6 @@ function mapDispatchToProps(dispatch: Dispatch): object {
 
 // @ts-ignore
 const IntlAdaptiveCard = injectIntl(AdaptiveCard);
-export default connect(mapStateToProps, mapDispatchToProps)(IntlAdaptiveCard);
+// @ts-ignore
```

Comment:
can we comment why we are ignoring something

Reply:
Addressed in next PR